.gitignore

```diff
@@ -35,8 +35,9 @@ hercules/local_amr_wind_demo/sample_copy.nc
 #Ignore csv files
 *.csv
 !tests/test_inputs/*.csv
-!example_case_folders/10_heterogeneous_wind/floris_standin_data.csv
-!example_case_folders/10_heterogeneous_wind/wind_power_reference_data.csv
+!example_case_folders/10_heterogeneous_wind/inputs/floris_standin_data.csv
+!example_case_folders/10_heterogeneous_wind/inputs/wind_power_reference_data.csv
+!example_case_folders/07_floris_standin_and_solar_pysam/inputs/Flatirons_solar_data_precursor-tztest.csv
```

Comment:
Seems like we also need to add `'!example_case_folders/07_floris_standin_and_solar_pysam/inputs/Flatirons_solar_data_highDNI.csv` here, if I'm understanding the issue correctly?